pluginlib/test/package.xml

```diff
@@ -1,14 +1,18 @@
 <?xml version="1.0"?>
```

Comment:
Assuming this manifest file is only used for testing and to find the plugin I would suggest to remove most of its content: dummy name, dummy version, specific description, no authors, dependencies, etc.